Library/Homebrew/dependency_collector.rb

```diff
@@ -170,13 +178,14 @@ def resource_dep(spec, tags)
 
   def parse_url_spec(url, tags)
     case File.extname(url)
-    when ".xz"          then xz_dep_if_needed(tags)
-    when ".zip"         then unzip_dep_if_needed(tags)
-    when ".bz2"         then bzip2_dep_if_needed(tags)
-    when ".lha", ".lzh" then Dependency.new("lha", tags)
-    when ".lz"          then Dependency.new("lzip", tags)
-    when ".rar"         then Dependency.new("unrar", tags)
-    when ".7z"          then Dependency.new("p7zip", tags)
+    when ".xz"              then xz_dep_if_needed(tags)
+    when ".zst", ".tar.zst" then zstd_dep_if_needed(tags)
```

Comment:
Does `.tar.zst` ever get matched here?

Reply:
You're right, I've confused `File::extname` with `Pathname#extname` here.
Thanks!